### ISSUE TITLE
Song crash sometimes

### ISSUE THREAD

**Issue by kijk2869** [kijk2869/discodo#39]
```[2020-08-18 13:48:41,774] [Loader.py] [libav.matroska,webm:Loader] [ERROR]: File ended prematurely```

may be occured in Stream ip tunneling.

**Comment by kijk2869** [comment]
Cannot reproduce in windows 10

**Comment by kijk2869** [comment]
may be sanic bug https://github.com/huge-success/sanic/pull/1876

**Comment by kijk2869** [comment]
```python
Traceback (most recent call last):
  File "C:\Users\mary\AppData\Local\Programs\Python\Python38\lib\site-packages\sanic\app.py", line 1041, in handle_request
    await response.send(end_stream=True)
  File "C:\Users\mary\AppData\Local\Programs\Python\Python38\lib\site-packages\sanic\response.py", line 102, in send
    await super().send(*args, **kwargs)
  File "C:\Users\mary\AppData\Local\Programs\Python\Python38\lib\site-packages\sanic\response.py", line 60, in send
    await self.stream.send(data, end_stream=end_stream)
  File "C:\Users\mary\AppData\Local\Programs\Python\Python38\lib\site-packages\sanic\http.py", line 281, in http1_response_normal
    raise ServerError("Response was smaller than content-length")
sanic.exceptions.ServerError: Response was smaller than content-length
```

**Comment by kijk2869** [comment]
> ```python
> Traceback (most recent call last):
>   File "C:\Users\mary\AppData\Local\Programs\Python\Python38\lib\site-packages\sanic\app.py", line 1041, in handle_request
>     await response.send(end_stream=True)
>   File "C:\Users\mary\AppData\Local\Programs\Python\Python38\lib\site-packages\sanic\response.py", line 102, in send
>     await super().send(*args, **kwargs)
>   File "C:\Users\mary\AppData\Local\Programs\Python\Python38\lib\site-packages\sanic\response.py", line 60, in send
>     await self.stream.send(data, end_stream=end_stream)
>   File "C:\Users\mary\AppData\Local\Programs\Python\Python38\lib\site-packages\sanic\http.py", line 281, in http1_response_normal
>     raise ServerError("Response was smaller than content-length")
> sanic.exceptions.ServerError: Response was smaller than content-length
> ```

Occurs due to streaming interruption, ignored

**Comment by kijk2869** [comment]
fix in #44
